studio/src/editors/merch-card-editor.js

```diff
@@ -685,13 +739,31 @@ class MerchCardEditor extends LitElement {
                                         "
                                                         ></div>
                                                     `
-                                                  : nothing}
+                                                  : isSpecialValue(color)
+                                                    ? html`
+                                                          <div
+                                                              style="
+                                            width: 16px;
```

Comment:
please do not use inline styles. Use regular CSS with variables whose values are set by the code. This will help to keep the JS code clean.

Reply:
Fixed